.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: CI для C++ та Arduino
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  cpp-arduino-build:
+    runs-on: windows-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      # Забезпечити налаштування Arduino CLI (якщо ще не доступний на ранері)
```

Comment:
?? What happened with encoding?

Reply:
have no idea actually
looks normal on my side in the branch

Reply:
i can change all comments to english if needed, perhaps there is some issue with ukrainian

Reply:
even here it looks normal btw
![1](https://github.com/user-attachments/assets/2312b0a9-dc70-438e-9e9c-12ce3bbf551e)